src/cswinrt/strings/additions/Windows.Storage.Streams/WindowsRuntimeBufferExtensions.cs

```diff
@@ -248,6 +249,32 @@ public static void CopyTo(this IBuffer source, uint sourceIndex, IBuffer destina
 
 #endregion (IBuffer).CopyTo extensions for copying to an (IBuffer)
 
+        public static bool TryGetDataUnsafe(this IBuffer buffer, out IntPtr dataPtr)
+        {
+            if (buffer == null)
+            {
+                dataPtr = IntPtr.Zero;
+                return false;
+            }
+
+            if (ComWrappersSupport.TryUnwrapObject(buffer, out var objRef))

```

Comment:
We lost the QI to IBufferByteAccess in the conversion?

Reply:
Yup will add back.